gpu/octree/src/cuda/radius_search.cu

```diff
@@ -75,33 +75,23 @@ namespace pcl
         struct IndicesQuery : public DirectQuery
         {
             const int* queries_indices;
-            __device__ __forceinline__ float3 fetch(int query_index) const
+            __device__ __forceinline__ float3 fetch(const int query_index) const
             {
-                PointType q = queries[queries_indices[query_index]];
+                const PointType q = queries[queries_indices[query_index]];
                 return make_float3(q.x, q.y, q.z);
             }
         };
 
         struct SharedRadius
         {
             float radius;
-            __device__ __forceinline__ float getRadius(int /*index*/) const { return radius; }
-            __device__ __forceinline__ float bradcastRadius2(float* /*ptr*/, bool /*active*/, float& /*radius_reg*/) const
-            {
-                return radius * radius;
-            }
```

Comment:
is this public API?

Reply:
It's a .cu file (similar to cpp) so I'd say no

Reply:
It's only used internally. Also a `__device__` function cannot be public API right? Because it can only be called within a kernel?